packages/odyssey-icons/README.md

```diff
@@ -0,0 +1,11 @@
+# `@okta/odyssey-icons`
+
+> TODO: description
+
+## Usage
+
+```
+const odysseyIcons = require('@okta/odyssey-icons');
```

Comment:
Do we want to show ES6 syntax instead?

Reply:
I plan on removing all of this - generated by `lerna create`, but I figured we could keep the initial til the real docs go in (this PR is mainly to get an environment ready for JJ to drop icons into).

Reply:
Can you please create a GH issue for this? Before this package is published let's make sure docs are available.

Reply:
@jmelberg-okta https://github.com/okta/odyssey/issues/572 - none of this is getting released until Icons are on lock

Reply:
Thanks for updating that issue!